tox.ini

```diff
@@ -12,5 +13,5 @@ setenv =
 # The standard library unittest module can run tests on Python 2.7 and newer
 commands =
     python -c "import OpenSSL.SSL; print(OpenSSL.SSL.SSLeay_version(OpenSSL.SSL.SSLEAY_VERSION))"
-    coverage run --branch --source=OpenSSL setup.py test
+    coverage run --branch --source=OpenSSL -m pytest OpenSSL
```

Comment:
Do you want to leave the test suite information in `setup.py` even though it's no longer being used by the CI system?